sdcm/nemesis.py

```diff
@@ -1737,6 +1739,18 @@ def disrupt(self):
         self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
 
 
+class GeminiNonDisruptiveChaosMonkey(Nemesis):
+    def __init__(self, *args, **kwargs):
+        super(GeminiNonDisruptiveChaosMonkey, self).__init__(*args, **kwargs)
+        run_with_gemini = set(self.get_list_of_disrupt_methods_for_nemesis_subclasses(run_with_gemini=True))
+        non_disruptive = set(self.get_list_of_disrupt_methods_for_nemesis_subclasses(disruptive=False))
+        self.disrupt_methods_list = run_with_gemini.intersection(non_disruptive)
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
+
+
 RELATIVE_NEMESIS_SUBCLASS_LIST = [NotSpotNemesis]
 
```

Comment:
Need to add GeminiNonDisruptiveChaosMonkey to COMPLEX_NEMESIS list

Reply:
fixed